content/blog/configuring-sso-for-hpe-greenlake-central-private-cloud-enterprise-and-hpe-greenlake-glcp-using-okta.md

```diff
@@ -0,0 +1,166 @@
+
+   See here for IdP attribute details: <https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us>
+
+   A new SAML attribute has been added “hpe_ccs_attribute” which tells HPE GreenLake and HPE Greenlake Central (Private Cloud Enterprise) the exact role/permissions for each user. The following describes how to format the attribute.
+
+   Format: {version}#{pcid}:{app id}:{role_name}:{ALL_SCOPES}
+
+![](/img/hpe-greenlake-saml-attributes.jpg)
+
+![](/img/workspace-pcid.jpg)
+
+![](/img/glp_role_name.jpg)
+
+The **hpe_ccs_attribute** always starts with version*1#. You must first configure the attributes for HPE GreenLake and To do so, enter the PCID for the account, followed by the HPE GreenLake application ID. This will always be **00000000-0000-0000-0000-000000000000**. Following this, enter the role name and ALL*SCOPES**.** 
+
+Example:
+
+version_1#7ede5c36b7b911edacf45a78eb8b07d1:00000000-0000-0000-0000-000000000000:Observer:ALL_SCOPES
+
+![](/img/saml_settings.jpg)
+
+2﻿. Complete the setup.
+
+![](/img/ws-image7.png)
+
+Click Next and Select “Internal App”, then Finish.
+
+**Step 3:** **Export the SAML 2.0 IdP metadata**
+
+1. Click Next – Configure the Sign On settings
+
+   You will find two options are available: **View Setup Instructions** which steps you through the SAML configuration and **Identity Provider metadata**, which will produce an XML file that can be loaded into HPE GreenLake Cloud Platform (GLCP) application
+
+   Suggestion: Click **Identity Provider metadata** and save the XML data to a file.
+
+   ![](/img/ws-image9.png)
+2. C﻿lick Next.       
+3. Select Internal app, and Click Finish.    
+
+   **Step 3.1 :  Access to the SAML application and HPE Greenlake is determined by assigning only those members or group to the SAML application.**
+
+![](/img/customer-user-assignment-to-saml.jpg)
+
+
+
+**S﻿teps to be followed in HPE GreenLake Cloud Platform(GLCP)**
+
+ **   Step 4: Create SAML Authorization Profile in HPE GreenLake Cloud Platform**
+
+1. Log into **HPE GreenLake Cloud Platform** and click Menu > Manage > Authentication and Click Set Up SAML Connection.
+
+   *Before you can add a new SAML configuration, you must have at least one user account with that domain already enabled in HPE GreenLake. Also, you must be logged into HPE GreenLake with an account from that domain in order to enable SSO for it.*
+
+   ![](/img/ws-image10.png)
+2. Type in the domain you want to enable SSO on:
+
+   ![](/img/glp_domain.jpg)
+3. Input the metadata from the step above.
+
+   While HPE GreenLake does support entering this information manually, it's recommended that you simply upload the XML metadata that was downloaded in the previous step. To do so, Select Metadata File, selecting the XML file. Then, click Next.
+
+   ![](/img/ws-image12.png)
+4. Enter the SAML attributes to match what was entered in Okta. Set the idle timeout value as well.
+
+   ![](/img/config_setting_sso_appjpg.jpg)
+5. Then click Next.    
```

Comment:
For bullets 5 and 6, there's a similar formatting issue:

![image](https://github.com/hpe-dev-incubator/hpe-dev-portal/assets/77496261/bce0c92f-c045-4a70-a5fc-123297514801)

By the way, should it be "Create a recover user" or "Create a recovery user"? In the image, it says "recovery".